src/tests/complex_tests/mod.rs

```diff
@@ -60,7 +60,9 @@ const EIP4844_CYCLE_LIMIT: usize = 4096;
 #[test]
 fn basic_test() {
     let test_artifact = read_basic_test_artifact();
-    run_and_try_create_witness_inner(test_artifact, 20000, None);
+    //run_and_try_create_witness_inner(test_artifact, 20000, None);
+    let blob = vec![3u8; 80000];
+    run_and_try_create_witness_inner(test_artifact, 20000, Some([blob.clone(), blob.clone()]));
```

Comment:
I run this test over a "fresh" directory - so that all proofs were recomputed. And it did generate the proof succesfully. Do you remember which part of the 4844 circuit would assert ?